cirq-ft/cirq_ft/algos/prepare_uniform_superposition.py

```diff
@@ -0,0 +1,98 @@
+# Copyright 2023 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Sequence, Tuple
+
+import attr
+import cirq
+import numpy as np
+from cirq._compat import cached_property
+from cirq_ft import infra
+from cirq_ft.algos import and_gate, arithmetic_gates
+
+
+@attr.frozen
+class PrepareUniformSuperposition(infra.GateWithRegisters):
+    r"""Prepares a uniform superposition over first $n$ basis states using $O(log(n))$ T-gates.
+
+    Performs a single round of amplitude amplification and prepares a uniform superposition over
+    the first $n$ basis states $|0>, |1>, ..., |n - 1>$. The expected T-complexity should be
+    $10 * log(L) + 2 * K$ T-gates and $2$ single qubit rotation gates, where $n = L * 2^K$.
+
+    However, the current T-complexity is $12 * log(L)$ T-gates and $2 + 2 * (K + log(L))$ rotations
+    because of two open issues:
+        - https://github.com/quantumlib/cirq-qubitization/issues/233 and
+        - https://github.com/quantumlib/cirq-qubitization/issues/235
+
+    Args:
+        n: The gate prepares a uniform superposition over first $n$ basis states.
+        cvs: Control values for each control qubit. If specified, a controlled version
```

Comment:
This says cvs but the arg is cv.  Also, I think we should have this be "control_values" rather than cv.

Reply:
Changed to `cv`. See my comment above regarding `control_values` vs `cv`. Tl;Dr former is too verbose and we are trying to be consistent with `cv` throughout the cirq_ft and bloqs packages.